### ISSUE TITLE
should we support the red high temperature screen type?

### ISSUE THREAD

**Issue by nyaray** [pappersverk/inky#7]
LUTs can be found here https://github.com/pimoroni/inky/blob/master/library/inky/inky.py#L172-L185

**Comment by lawik** [comment]
Never seen it. We could, at some point. It would be nice if one of us had it in that case.

**Comment by nyaray** [comment]
Agreed, I'll put it on hold until that is the case or someone requests it.
